addon/styles/themes/blue-sky/elements/icon.overrides

```diff
@@ -61,6 +61,10 @@ i.icon.calendar:before { content: "\e81c"; color: @colorIcon; font-family: 'Crim
 i.icon.check-square-o:before { content: "\f046"; font-size: 20px;}
 i.icon.check-all-square-o:before { content: "\e815"; font-family: 'Crimean'; font-size: 20px;}
 
+/* Site map */
```

Comment:
Why define additional classes for these icons? They already exist in the set of [`Semantic UI` interface icons](https://semantic-ui.com/elements/icon.html#interfaces).